robottelo/vm.py

```diff
@@ -232,11 +234,13 @@ def register_contenthost(self, activation_key, org):
             registration.
 
         """
-        return self.run(
-            u'subscription-manager register --activationkey {0} '
-            '--org {1} --force'
-            .format(activation_key, org)
-        )
+        cmd = (u'subscription-manager register --activationkey {0} '
+               '--org {1} '
```

Comment:
This looks ok but as it is building the command incrementally makes more sense to drop trailing space and every time a new option is appended add the space before the new option.